doc/src/custom/numfig.py

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
This extension hasn't been updated since 2012 which makes it a little high risk. If we are lucky this functionality might be provided in a more contemporary version of Sphinx. The Cylc tutorial requires 1.5.3 so we will have to up this anyway.

If not it looks like someone has put it up on pypi which would at least save us bundling the script: https://pypi.org/project/sphinx_numfig/

Reply:
[Addressing in latest comment; see :high_brightness:.]